mitodl/open-discussions
# restyle LR drawer

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #2466
closes #2253

#### What's this PR do?

This updates the display of learning resources in the drawer to match our latest designs. I also updated the display of the logo for the LR platform.

I have a separate PR open (#2484) which fixes the `TruncatedText` component.

Basically, this mainly is just restyling things, but there are a few more major parts:

- I pulled out hte styling for the expanded learning resource display from `course.scss` to a new file, `learning-resource-drawer.scss`, which has only the styling for the drawer in it.
- I also finally updated a lot of selectors like were of the form `course-foo` to be just `foo`, since those selectors were originally written when we were only showing courses and we're now showing bootcamps and programs and so on.

#### How should this be manually tested?

look at all kinds of learning resources and make sure that nothing is broken. make sure that the design is good based on the discussion on the issue, that things are readable on all screen widths and so on.

#### Screenshots (if appropriate)

![ffdasdfbootcampasdfasfdvideoasdfasdf](https://user-images.githubusercontent.com/6207644/70832103-47948c80-1dc2-11ea-91d1-48ac34cc1940.png)
![ffdasdfbootcampasdfasfdvideo](https://user-images.githubusercontent.com/6207644/70832104-47948c80-1dc2-11ea-90e9-8a38a3172cf1.png)
![ffdasdfbootcampasdfasfd](https://user-images.githubusercontent.com/6207644/70832105-47948c80-1dc2-11ea-9ac4-ddfcfdfd571b.png)
![ffdasdfbootcamp](https://user-images.githubusercontent.com/6207644/70832106-482d2300-1dc2-11ea-91e5-7015bc4eb512.png)
 ⛔ 🚧 🛑 📣
